_data/hosts.yml

```diff
@@ -70,13 +70,9 @@
     name: Aruba.it
     url: 'http://hosting.aruba.it/'
     type: shared
-    default: 5.5.28
+    php70: 0
```

Comment:
This php70 is not meant to be here. If you saw one anywhere else its an artifact of a merge that I didn't notice.

If you offer PHP 7.0 would you add it to the main versions array below?